.github/workflows/gh_registry.yml

```diff
@@ -24,6 +53,43 @@ jobs:
       name: ghcr-deployment
       url: ${{ vars.deployment_url }}
 
+    strategy:
+      matrix:
+        image: ${{ fromJson(needs.ghcr_config.outputs.external_images).images }}
+      fail-fast: false
+
+    steps:
+      - name: Login to GitHub CR
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ github.token }}
```

Comment:
The actor may change, but that's fine/good. The only thing that is relevant are the permissions of the github token. Any use of PATs at this point I think is mostly superseded by the gh token and should be avoided when possible. The github token here is in fact the only token that would be permitted to push the images, since I personally can't push images without using a GitHub workflow (nor can anyone else).